### ISSUE TITLE
Improvements for sim-a11y-view.html template

### ISSUE THREAD

**Issue by samreid** [phetsims/chipper#658]
I noticed a few issues in the sim-a11y-view.html template.  I'll mark them here as checkboxes, then assign to @jessegreenberg for work.  After the work is complete, the actual sim a11y HTMLs should be regenerated and committed.  This was discovered during https://github.com/phetsims/friction/issues/64

- [x] Typo in `ineficient`
- [x] `intervalId` is never assigned to a var, and hence is getting assigned to window as a global.
- [x] typo in the last word in this phrase: `// set focus to the loaded ifram`
- [x] variable is never used: `var politeElementContainer = document.getElementById( 'polite-element-container' );`

**Comment by jessegreenberg** [comment]
Thanks @samreid, fixes made in the above commits. I regenerated a11y views with
`perennial/bin/for-each.sh accessibility grunt generate-a11y-view-html` and pushed. Closing this issue.
